translations/translations.qrc

```diff
@@ -3,5 +3,7 @@
         <file>app_en.qm</file>
         <file>app_fr.qm</file>
         <file>app_zh.qm</file>
+        <file>app_pt.qm</file>
+        <file>app_es.qm</file>
```

Comment:
Please also add them into https://github.com/Inokinoki/QEFIEntryManager/blob/959673173c5723f2b0881edec5332f196475f859/CMakeLists.txt#L50

Reply:
Now included in CMakeLists.txt